frontend/src/scenes/cohorts/cohortLogic.ts

```diff
@@ -162,20 +165,17 @@ export const cohortLogic = kea<cohortLogicType<CohortLogicProps>>({
                         return values.cohort
                     }
                 },
-                saveCohort: async ({ cohortParams, filterParams }, breakpoint) => {
-                    let cohort = { ...values.cohort, ...cohortParams }
```

Comment:
Hm what is the reason we don't extend from the old cohort state?

Reply:
We could, but we don't need to because:

1. The API is additive, and doesn't remove fields you don't send
2. Kea-forms onSubmit submits the entire object, so `cohortParams` effectively always have the entire 'current' cohort

Reply:
makes sense 👍